.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: java
+jdk:
+  - oraclejdk8
+sudo: required
+services:
+  - docker
+
+before_install:
```

Comment:
these lines are only needed for testing of "the latest and greatest" Docker. I think you can remove the whole "before_install" block